packages/sse/lib/sse-service.js

```diff
@@ -0,0 +1,279 @@
+const IncomingMessage = require('http').IncomingMessage;
+const ServerResponse = require('http').ServerResponse;
+const EventEmitter = require('events');
+const uuidv4 = require('uuid/v4');
+const async = require('async');
+const assert = require('./utils/assert');
+const { maybeFn } = require('./utils/maybe');
+const { SSEIDClassFactory } = require('./sse-id');
+
+const internal = require('./utils/weak-map').getInternal();
+
+const HEARTBEAT_INTERVAL = 15;
+const HEARTBEAT_MESSAGE = 'heartbeat';
```

Comment:
I feel the default heartbeat should be fairly short - a lot of clients could easily cause quite a lot of traffic being pinged every 15 seconds.

Reply:
The choice of `15` directly comes from [the specification](https://html.spec.whatwg.org/multipage/server-sent-events.html#authoring-notes).

I am a bit confused by your comment, as you seem to imply that I should shorten the interval, but at the same time doing so would bring even more traffic (which you apparently whishes to avoid). 

Or maybe you were simply suggesting that we sending all heartbeat at once might be problematic for performance ?

Reply:
Sorry, I actually meant the message length! As far as I know, we could potentially just send an empty comment, which would reduce each ping from 12 to 3 bytes.

Reply:
Fair enough. 

For a very large number of connections this is not negligible, you're right. I'm changing it.